### PR TITLE
Remove unnecessary artifact properties

### PR DESCRIPTION
**Pull request by sarah256**
Relates to JIRA: CLOUDWF-556

### CODE REVIEW

File: estuary_updater/handlers/koji.py [hunk 1]
```diff
@@ -29,8 +28,6 @@ def can_handle(msg):
             '/topic/VirtualTopic.eng.brew.build.failed',
             '/topic/VirtualTopic.eng.brew.build.canceled',
             '/topic/VirtualTopic.eng.brew.build.deleted',
-            '/topic/VirtualTopic.eng.brew.build.tag',
-            '/topic/VirtualTopic.eng.brew.build.untag'
```
Comment (mprahl) [inline estuary_updater/handlers/koji.py:33]: Could you please also remove the topics in `fedmsg.d/example.py`?

File: tests/handlers/test_koji.py [hunk 2]
```diff
@@ -183,30 +175,6 @@ def test_build_deleted(mock_koji_cs):
     assert build.state == 2
 
 
-def test_build_tag(kb_one):
-    """Test the Koji handler when it recieves a new build tag message."""
-    with open(path.join(message_dir, 'koji', 'build_tag.json'), 'r') as f:
```
Comment (mprahl) [inline tests/handlers/test_koji.py:188]: Could you please delete build_tag.json since it's not longer used?

File: tests/handlers/test_koji.py [hunk 3]
```diff
@@ -183,30 +175,6 @@ def test_build_deleted(mock_koji_cs):
...
-    handler.handle(msg)
-
-    tag = KojiTag.nodes.get_or_none(id_=15638)
-
-    assert tag.builds.is_connected(kb_one)
-
-
-def test_build_untag(kb_one, koji_tag):
-    """Test the Koji handler when it recieves a new build untag message."""
-    with open(path.join(message_dir, 'koji', 'build_untag.json'), 'r') as f:
```
Comment (mprahl) [inline tests/handlers/test_koji.py:201]: Could you please delete build_untag.json since it's not longer used?

File: tests/conftest.py [hunk 4]
```diff
@@ -152,15 +151,6 @@ def kb_one():
...
-def koji_tag():
-    """Return a KojiTag."""
-    return KojiTag.get_or_create({
-        'id_': '15638',
-        'name': 'rhos-13.0-rhel-7-pending'
-    })[0]
-
-
 @pytest.fixture
 def module_build_getTag():
```
Comment (mprahl) [inline tests/conftest.py:155]: This doesn't seem to be used anymore. Could this also be removed?

File: estuary_updater/handlers/koji.py [hunk 5]
```diff
@@ -75,52 +68,21 @@ def build_handler(self, msg):
...
-
-                    module_build_tag.module_builds.connect(build)
-
                     _, components = self.koji_session.listTaggedRPMS(module_build_tag_name)
                     for component in components:
                         component_build = self.get_or_create_build(component)
                         build.components.connect(component_build)
-            elif build.__label__ == ContainerKojiBuild.__label__:
+
+            if build.__label__ == ContainerKojiBuild.__label__:
```
Comment (mprahl) [inline estuary_updater/handlers/koji.py:83]: Why did you change this from `elif` to `if`?

File: tests/handlers/test_distgit.py [hunk 6]
```diff
@@ -67,31 +57,6 @@ def test_distgit_commit():
     assert not commit.reverted_bugs.all()
 
 
-def test_distgit_push():
-    """Test the dist-git handler when it recieves a new push message."""
-    with open(path.join(message_dir, 'distgit', 'distgit_push.json'), 'r') as f:
```
Comment (mprahl) [inline tests/handlers/test_distgit.py:72]: Could you please remove `distgit_push.json` since it's no longer used?

File: estuary_updater/handlers/distgit.py [hunk 7]
```diff
@@ -25,8 +25,7 @@ def can_handle(msg):
         :rtype: bool
         """
         supported_topics = [
-            '/topic/VirtualTopic.eng.distgit.commit',
-            '/topic/VirtualTopic.eng.distgit.push'
```
Comment (mprahl) [inline estuary_updater/handlers/distgit.py:29]: Could you also remove this from `fedmsg.d/example.py`?

Comment (sarah256) [reply estuary_updater/handlers/distgit.py:29]: For whatever reason it's already not in `example.py`, just in case you're looking for this change
